docs/outlook/onmessagesend-onappointmentsend-events.md

```diff
@@ -147,7 +147,12 @@ When Outlook launches without internet connectivity, it's unable to determine wh
 
 # [Web/Windows (new)](#tab/web-new-windows)
 
-For Outlook on the web and new Outlook on Windows, configure the **OnSendAddinsEnabled** mailbox policy in Exchange Online PowerShell. This ensures that outgoing messages are checked before they're sent.
+> [!NOTE]
+>
+> - We're working on deploying the **OnSendAddinsEnabled** mailbox policy. Periodically check this article for updates.
```

Comment:
Where is this being deployed? Can you clarify what this means that it's being deployed?